src/julia/Float.jl

```diff
@@ -91,7 +91,7 @@ divexact(a::Rational{T}, b::AbstractFloat; check::Bool=true) where T <: Union{Si
 divexact(a::Rational{BigInt}, b::BigFloat; check::Bool=true) = a/b
 
 function divides(a::BigFloat, b::BigFloat)
```

Comment:
Off-topic, but: why do we define this only for `BigFloat`, not for `AbstractFloat`? (or alternatively, why do we define this at all... but I guess that ship has sailed *sigh*)